community/zathura-pdf-mupdf/depends

```diff
@@ -0,0 +1,8 @@
+cairo
+jbig2dec
+libjpeg-turbo
+libmupdf make
```

Comment:
Only a `make` dep?

Reply:
Yes. As far as libraries go, `libmupdf` only contains object archives and headers. I've tested this.

Reply:
:+1: